PyInstaller/depend/imphook.py

```diff
@@ -403,8 +406,12 @@ def _load_hook_module(self):
         head, tail = os.path.split(self.hook_filename)
         logger.info(
             'Loading module hook %r from %r...', tail, head)
-        self._hook_module = importlib_load_source(
-            self.hook_module_name, self.hook_filename)
+        try :
+            self._hook_module = importlib_load_source(
+                self.hook_module_name, self.hook_filename)
+        except ImportError:
+            raise Exception(self.exceptions.hook_import_fail())
```

Comment:
You should be raising your custom exception type here - not `Exception`.

Reply:
This is not how exceptions work. You want to simply 
```suggestion
            raise ImportErrorInHook(self.module_name, self.hook_filename)
```